README.md

```diff
@@ -110,3 +110,19 @@ contract TransactionBatcher {
     }
 }
 ```
+
+## Environment Variables
+
+To run the bots in this repository, duplicate the `.env` file and rename the copy to either `.env.production` or `.env.staging`.
+
+Below is an example of what a .env file would look like for deployments on the Kovan network.
+
+```
+WEB3_PROVIDER_URL=https://kovan.infura.io/v3/<api-key>
+TRANSACTION_BATCHER_CONTRACT_ADDRESS=0x741A4dCaD4f72B83bE9103a383911d78362611cf
+BATCH_SEND_PRIVATE_KEY=0x2E28195A4F392FC380024636A0D14980F726849CC55024A01BF842D16605CFEB
+KLEROS_LIQUID_CONTRACT_ADDRESS=0x60b2abfdfad9c0873242f59f2a8c32a3cc682f80
+MULTIPLE_ARBITRABLE_TRANSACTION_CONTRACT_ADDRESS=
+T2CR={"address": "0x25dd2659a1430cdbd678615c7409164ae486c146","blockNumber": 10452223}
+BADGE_TCRS=[{"address": "0xd58bdd286e8155b6223e2a62932ae3e0a9a75759","blockNumber": 10452276},{"address": "0x78895ec026aeff2db73bc30e623c39e1c69b1386","blockNumber": 10708158}]
```

Comment:
Have you tested this? I was under the impression that you can't have arrays as env variables

Reply:
You are correct that we can't have arrays and objects as env variables, but these strings, later parsed with `JSON.parse()`